src/features/unmarkedWsNoticeList/utils.ts

```diff
@@ -93,3 +110,51 @@ export const getDraftedOffers = (applications: UnmarkedWinterStorageNotice[]) =>
       },
     ];
   }, []);
+
+export const generateAndSaveStickerPDF = (customers: CustomerInfo[]) => {
```

Comment:
First attempt to generate the pdf was to use jsPDF's html support and directly rendering a react component's markup. Due to several issues with the output (css units and typesetting being the main ones), the best way to achieve the desired output specified in the designs was to do this programmatically though the library's API.